web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,162 @@
+
+Click the **Upgrade** button, and change the site plan to **Performance** > **Medium**. 
+
+** Clone your sites repo on your local
+
+** Change into the newly cloned directory
+
+** Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
+
+** Checkout the main branch from upstream:
+
+```
+
+git fetch upstream main
+
+git checkout -t upstream/main
+
+```
+
+** Reset the master branch to match main, then force push that to the Dev environment:
+
+```
+
+git branch -D master
+
+git checkout -b master
+
+git push origin master --force
+
+```
+
+:::Note
+Make sure code has synced and all active workflows are complete in the UI.
+Click on the **Visit Development Site** button to Install via the UI—selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
+:::
+
+## Installing using Build Tools
+### Prerequisites
 
 - Composer (required for CMS backends): [Install Globally](https://getcomposer.org/download/)
 - [Generate machine token](https://pantheon.io/docs/machine-tokens#create-a-machine-token) & [Authenticate into Terminus](https://pantheon.io/docs/machine-tokens#authenticate-into-terminus)
 - [Install Terminus](https://pantheon.io/docs/terminus/install) (3.0.0 above required)
 - Also install the following plugins:
   - `terminus self:plugin:install terminus-build-tools-plugin`
-  - `terminus self:plugin:install terminus-power-tools`
+  - `terminus self:plugin:remove terminus-power-tools`
   - `terminus self:plugin:install terminus-secrets-plugin`
   - Reload the terminus plugins: `terminus self:plugin:reload`
   - Clear cache for composer: `composer clear-cache`
   - Validate that the required plugins are installed: `terminus self:plugin:list`
 - Create [Github Personal access tokens](https://github.com/settings/tokens)
 - Create [CircleCI Personal API Tokens](https://app.circleci.com/settings/user/tokens)
+- Create [Bitbucket app password](https://bitbucket.org/account/settings/app-passwords/)  for your [username](https://bitbucket.org/account/settings/username/change/)
+- Create [GitLab Personal API Tokens](https://gitlab.com/-/profile/personal_access_tokens)
 
-## Installation
+### Installation
+1.  Run the `terminus build:project:create` as follows for Github with CircleCI: 
+  ```
+  terminus build:project:create \
+    --team='Pantheon Employees' \
+    --template-repository="git@github.com:pantheon-systems/decoupled-drupal-composer-managed.git" pantheon-upstreams/decoupled-drupal-composer-managed \
+    --visibility private {PROJECT_NAME} \
+    --profile="pantheon_decoupled_profile" \
+    --stability=dev
+  ```
+
+  Replace `{PROJECT_NAME}` with your project name - for example `decoupled-drupal`.
+
+  Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+2. Run the `terminus build:project:create` as follows for Github with Github Actions:
```

Comment:
Rather than repeating this four times, could we instead use one generic example and then include the 'Additional options` section from the existing documentation?